ios/PurchasesHybridCommon/PurchasesHybridCommon/CommonFunctionality.swift

```diff
@@ -308,10 +308,10 @@ import RevenueCat
         }
     }
 
-    @objc(paymentDiscountForProductIdentifier:discount:completionBlock:)
-    static func paymentDiscount(for productIdentifier: String,
-                                discountIdentifier: String?,
-                                completion: @escaping ([String: Any]?, ErrorContainer?) -> Void) {
+    @objc(promotionalOfferForProductIdentifier:discount:completionBlock:)
+    static func promotionalOffer(for productIdentifier: String,
+                                 discountIdentifier: String?,
+                                 completion: @escaping ([String: Any]?, ErrorContainer?) -> Void) {
```

Comment:
can you update line 331 variable name to be `promotionalOfferCompletion`?

Reply:
I'm confused - why is this needed? We don't do it for any of the other completions. 
Note that this is promotionalOffers, not Promoted IAPs, i.e.: not the ones that get deferred. 
So this is just a regular completion block

Reply:
Sorry for the absurdly late reply btw

Reply:
Just to be clear, I am talking about line 331. https://github.com/RevenueCat/purchases-hybrid-common/pull/133/files#diff-463b908f62c781c1e5d798d18af1016da664f05a238692e5181c0044e15ee2f9R331

Reply:
oohhh okay, thanks!! You're totally right, I'll update it

Reply:
done!